website/docs/cloud-docs/vcs/troubleshooting.mdx

```diff
@@ -82,7 +82,7 @@ Note that the "SSH Key" section in a workspace's settings is only used for mid-r
 
 ## General
 
-### Terraform Cloud returns 500 after authenticating with the VCS provider (other than Bitbucket Server)
+### Terraform Cloud returns 500 after authenticating with the VCS provider (other than Bitbucket Data Center)
```

Comment:
```suggestion
The following errors may occur for all VCS providers except Bitbucket Data Center.

### Terraform Cloud returns 500 after authenticating with the VCS provider 
```

Reply:
"Terraform Cloud returns 500 after authenticating with the VCS provider (other than Bitbucket Data Center)" is a distinct error. Should it not retain its h3 (`###`) status?